website_mrp_bom/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    'name': 'Website MRP BOM',
+    'category': 'Website',
+    'summary': 'Bill of Materials Module for Website',
+    'version': '11.0.1.0.0',
+    'description':
```

Comment:
I think the description will *cover* the README in the module's form

Reply:
I'm not understanding it

Reply:
@hveficent see [here](https://github.com/odoo/odoo/blob/11.0/odoo/modules/module.py#L337): when you set the `description` field in the manifest, the README is no more visible in the module's form